utils/storybook/StaticStatesTable.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {StaticStates} from '@workday/canvas-kit-labs-react-core/lib/StaticStates';
+
+interface ComponentPropValue {
+  value: any;
+  label: string;
+}
+
+interface ComponentProps {
+  [key: string]: any;
+}
+
+export interface StaticStatesTableProps {
+  /**
+   * The pseudo states + disabled state of the component. These correspond to the columns of the table.
+   * @default [ '', 'hover', 'focus', 'focus hover', 'active', 'active hover', 'disabled', 'disabled hover' ]
+   */
+  states: string[];
+  /**
+   * The props that will be passed to the component when it is rendered. The combinations of these will
+   * correspond to the rows of the table.
+   * @default {}
+   */
+  componentProps: {
+    [key: string]: ComponentPropValue[];
+  };
+  /**
+   * The render function called to render the component in each cell of the table. This gives you
+   * the ability to add extra styling or markup (a blue background for an inverse variant, for example).
+   */
+  renderComponent(props: ComponentProps, disabled: boolean, className: string): React.ReactNode;
+  /**
+   * The validation function called to check the validity of the current prop combination.
+   * Return false if you would like to skip this prop combination (table row).
+   * Return true otherwise.
+   * @default (props) => true
+   */
+  verifyProps(props: ComponentProps): boolean;
+}
+
+const Table = styled('table')({
+  width: '100%',
+  thead: {
+    textAlign: 'left',
+    paddingBottom: 16,
+  },
+  'td, th': {
+    minWidth: 100,
+    paddingBottom: 16,
+    paddingRight: 16,
+    textAlign: 'left',
+  },
+});
+
+export default class StaticStatesTable extends React.Component<StaticStatesTableProps> {
+  public static defaultProps = {
+    componentProps: {},
+    states: [
+      '',
```

Comment:
should we just add in default? or do we want empty here?

Reply:
Good point - I'm rewriting this to `Default` anyways. Will update.